### PR TITLE
Complete metrics splitting for activity model 

### PR DESCRIPTION
**Pull request by lishengbao**

### CODE REVIEW

File: compass_metrics/common.py
Comment (eyehwan) [inline compass_metrics/common.py:1]: I think this file should be merged into contributor_metrics

**Review by eyehwan — COMMENTED** [review]
minor comments.

**Review by EdmondFrank — APPROVED** [review]
LGTM

File: compass_metrics/common.py
Comment (lishengbao) [reply compass_metrics/common.py:1]: done!

**Review by eyehwan — APPROVED** [review]
LGTM!

**Comment by eyehwan** [comment]
@lishengbao You need to solve the conflict..

**Comment by lishengbao** [comment]
> @lishengbao You need to solve the conflict..

done!

**Pull request merged**